agent/exploits/cve_2024_7589.py

```diff
@@ -75,22 +74,13 @@ def _create_vulnerability(
     return vulnerability
 
 
-def _parse_version(version_string: str) -> str:
-    """Extracts the version part from the banner."""
-    match = re.search(r"OpenSSH_(\d+\.\d+(p\d+)?)", version_string)
-    return match.group(1) if match is not None else ""
-
-
 def _is_vulnerable_version(version_string: str) -> bool:
     """Checks if the version is vulnerable."""
-    version = _parse_version(version_string)
-    if version is not None:
-        if version in VULNERABLE_VERSIONS:
+    if version_string is not None:
```

Comment:
I would rename the arg to `ssh_banner` or `banner` ..

Reply:
It's used elsewhere I didn't want any confuse.